rust/stackable-cockpit/src/platform/cluster/resource_request.rs

```diff
@@ -11,6 +11,8 @@ use stackable_operator::{cpu::CpuQuantity, memory::MemoryQuantity};
 
 use crate::utils::k8s::{KubeClient, KubeClientError};
 
+const HELP_MESSAGE: &str = ". Have a look at https://github.com/torvalds/linux/blob/f7757129e3dea336c407551c98f50057c22bb266/include/math-emu/double.h#L29 for a possible solution";
```

Comment:
Please fix:

1. The tone
2. The message
3. The useless constant name

Reply:
This is more of an easteregg. But let me know what you want changed.

Reply:
Easter eggs in all their glory, but not when they insult the user or impede maintenance.

Reply:
I guess I will remove it then.